source/server/index.js

```diff
@@ -1,3 +1,8 @@
-import ServerApp from '/source/imports/application/server-application.js'
+import ServerApp from '../imports/application/server-application.js';
 
-ServerApp.start();
\ No newline at end of file
+Meteor.startup(function() {
+
+  if (Meteor.isServer) {
+    ServerApp.start();
```

Comment:
Exporting an app instance from the index seems like the right way to go, as this is essentially equivalent to the old namespace approach:

``` javascript
Meteor.startup(function() {
  if (Meteor.isServer) {
    Namespace.app = new ServerApp.start();
    Namespace.app.start();
  }
}
```